### ISSUE TITLE
Unable to populate ALT Text when an image is in the "content" of a set

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#120]
> **Migrated from GitLab**
> **Original Author:** Lenszie Vilfranche (`@vilfranchel`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/366

## Expected behaviour

ALT text field should be populating for images within complex item sets. 

## Current behaviour

When working on complex item sets the ALT text field does not populate/appears to be stuck. 

## Steps to reproduce

1. Recording: https://go.screenpal.com/watch/cZfjb7VstkZ

## Testing environment

* Username of the user who experienced the problem: Laken
* URL where the problem occured: 
* Browser name and version: Chrome
* User machine: Mac

## QA Steps 

[To be completed by Coko]

## Scheduling

[To be completed by Coko]

* Dependencies: [provide issue numbers]
* Development estimate: [provide in hours]
